src/ucm/cuda/cudamem.c

```diff
@@ -113,6 +115,30 @@ static void ucm_cuda_dispatch_mem_alloc(CUdeviceptr ptr, size_t length,
             ucm_warn("cuPointerSetAttribute(%p) failed: %s", (void*)ptr,
                      cu_err_str);
         }
+
+        event.mem_type.disable_reg = 1;
```

Comment:
maybe just do not shoot memtype alloc events on t4 by default?
add parameter
UCX_MEM_CUDA_MEM_ALLOC=y|n|auto
auto - yes on t4, no on others
yes - always
no - never